src/Solitaire.java

```diff
@@ -0,0 +1,24 @@
+import GUI.GamePanel;
+
+import javax.swing.*;
+import java.io.IOException;
+
+public class Solitaire extends JFrame {
+
+    static protected GamePanel gamePanel;
+    public static final int gamePanelWidth = 1080;
+    public static final int gamePanelHeight = 1920;
+
+    public Solitaire() throws IOException {
+        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        gamePanel = new GamePanel();
+        gamePanel.setSize(gamePanelWidth, gamePanelHeight);
+        add(gamePanel);
+        pack();
+    }
+
+
+    public static void main(String[] args) throws IOException {
```

Comment:
Avoid throwing in main
```suggestion
    public static void main(String[] args) {
        try {
            new Solitaire().setVisible(true);
        } catch (IOException e) {
            System.err.println("Caught IOException: " + e);
            e.printStackTrace();
        } catch (Exception e) {
            System.err.println("Caught unexpected exception: " + e);
            e.printStackTrace();
        }
```